installer/client/src/pages/CurrentVersionScreen.vue

```diff
@@ -0,0 +1,139 @@
+      versions listed in the dropdown selecting a different version would lead
+      to an upgrader failure and a database corruption.
+    </oxd-text>
+
+    <oxd-form-row class="orangehrm-current-version-row">
+      <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+        <oxd-grid-item>
+          <oxd-input
+            v-model="config.version"
+            type="select"
+            label="Current OrangeHRM Version"
+            :options="getItems"
+            :show-empty-selector="false"
+          ></oxd-input>
+        </oxd-grid-item>
+      </oxd-grid>
+    </oxd-form-row>
+
+    <Notice title="important">
+      <oxd-text tag="p">
+        If you have enabled data encrypted in your current version, you need to
+        copy the file 'lib/confs/key.ohrm' from your current installation to
+        corresponding location in the new version
+      </oxd-text>
+    </Notice>
+
+    <oxd-text tag="p" class="orangehrm-current-version-content">
+      Click <b>Next</b> to commence upgrading your instance
+    </oxd-text>
+
+    <oxd-form-actions class="orangehrm-current-version-action">
+      <oxd-button display-type="ghost" label="Back" @click="navigateUrl" />
+      <oxd-button display-type="secondary" label="Next" type="submit" />
+    </oxd-form-actions>
+  </oxd-form>
+</template>
+
+<script>
+import Notice from '@/components/Notice.vue';
+import InputField from '@ohrm/oxd/core/components/InputField/InputField.vue';
+import {APIService} from '@/core/util/services/api.service';
+import {navigate} from '@/core/util/helper/navigation.ts';
+
+export default {
+  name: 'CurrentVersionScreen',
+  components: {
+    Notice,
+    'oxd-input': InputField,
+  },
+  setup() {
+    const http = new APIService(
+      'https://8fdc0dda-8987-4f6f-9014-cb8c49a3a717.mock.pstmn.io',
+      'upgrader/current-version',
+    );
+    return {
+      http,
+    };
+  },
+  data() {
+    return {
+      config: {
+        version: '',
+      },
+      items: () => [],
+    };
+  },
+  computed: {
+    getItems() {
+      if (this.items.length == 0) {
+        return [];
+      }
+      return JSON.parse(JSON.stringify(this.items));
+    },
+  },
+  mounted() {
+    this.http.request({method: 'get'}).then((res) => {
+      this.items = res.data;
+    });
+  },
+  methods: {
+    navigateUrl() {
+      navigate('/upgrader/system-check');
+    },
+  },
+};
+</script>
+<style src="./installer-page.scss" lang="scss" scoped></style>
+<style lang="scss" scoped>
+.orangehrm-current-version {
+  height: 100%;
+  font-size: $oxd-input-control-font-size;
+  &-action {
+    padding: 1rem;
+    button {
+      margin-left: 0.5rem;
+    }
+  }
+  &-title {
+    color: $oxd-primary-one-color;
+    padding: 0 0.75rem 0.75rem 0.75rem;
+  }
+  &-content {
+    font-size: $oxd-input-control-font-size;
+    padding: 0.75rem;
+  }
+  ::v-deep(.oxd-grid-3) {
+    width: 100%;
+    margin: 0 !important;
```

Comment:
is this important needed? why do we set these for oxd-grid-3 ?

Reply:
important is not needed, but need to override width & margin